app/src/main/res/drawable/thumb_figma.xml

```diff
@@ -0,0 +1,21 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="200dp"
+    android:height="295.49dp"

```

Comment:
Would be best to keep under 256x256

Reply:
oops, that one slipped through. I'll fix it in an instant

Reply:
https://developer.android.com/studio/write/vector-asset-studio#when even recommends 200dp x 200dp